### PR TITLE
Add self to maintainers

### PR DESCRIPTION
**Pull request by philip-peterson**
Hi all,

I've very much enjoyed working on Gitea and was hoping to make it official by requesting maintainership. :) Excited to see what the future holds for this project.

My [merged PRs list](https://github.com/go-gitea/gitea/pulls?q=is%3Apr+author%3Aphilip-peterson+is%3Amerged+)

Cheers,
Phil

### CODE REVIEW

**Comment by wolfogre** [comment]
Thanks and welcome!

But I think there are two more things you should do before becoming a maintainer:

- Read the [CONTRIBUTING](https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md)
- image (😁I believe you will get it when you read the CONTRIBUTING)

**Comment by philip-peterson** [comment]
👍 sounds good, wasn't sure what the cutoff was. Maybe this PR can be closed until later?

**Comment by jolheiser** [comment]
More than happy to leave it open if you'd like, and thanks for the great work you've done thus far!

**Comment by philip-peterson** [comment]
Got my fourth today! https://github.com/go-gitea/gitea/pulls?q=is%3Apr+author%3Aphilip-peterson+is%3Amerged+

**Review by techknowlogick — APPROVED** [review]
Wooo!

**Review by jolheiser — APPROVED** [review]

**Review by delvh — APPROVED** [review]

File: MAINTAINERS [hunk 1]
```diff
@@ -51,3 +51,4 @@ Dong Ge <gedong_1994@163.com> (@sillyguodong)
 Xinyi Gong <hestergong@gmail.com> (@HesterG)
 wxiaoguang <wxiaoguang@gmail.com> (@wxiaoguang)
 Gary Moon <gary@garymoon.net> (@garymoon)
+Philip Peterson <philip.c.peterson@gmail.com> (@philip-peterson)
```
Comment (jolheiser) [inline MAINTAINERS]: Woops, removed the newline when I resolved this for you. 😅 
```suggestion
Philip Peterson <philip.c.peterson@gmail.com> (@philip-peterson)

```

**Comment by philip-peterson** [comment]
Thank you all! Excited to join the team 😄

**Comment by philip-peterson** [comment]
@techknowlogick Yes, it's philip#6132 , just sent you a message

**Pull request merged**